index.html

```diff
@@ -66,7 +66,7 @@ <h3>Friday <b>April 19th</b>, 2019 from <b>9 AM to 10PM</b> in the North Creek E
                 </h3>
                 <div class="register-buttons">
                     <a href="https://goo.gl/forms/DWhnBaRRVMHm0wTL2" class="top-fold__register">
```

Comment:
```suggestion
                    <a href="#" class="top-fold__register">
```
merge broke prev review